src/Components/OATGraphViewer/Internal/OATGraphCustomEdge.tsx

```diff
@@ -577,6 +577,8 @@ const OATGraphCustomEdge: React.FC<IOATGraphCustomEdgeProps> = ({
                                 setName={setNameText}
                                 dispatch={dispatch}
                                 state={state}
+                                relationship={nameEditor}
+                                relationshipClose={setNameEditor}
```

Comment:
Can we leverage the existing `onCommit` event here rather than introducing these two new props?